pipelines/internal/commands/run/run.go

```diff
@@ -364,7 +365,9 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		Environment: environment,
 	}
 
-	pipeline.Actions = []*genomics.Action{mkdir(directories)}
+	pipeline.Actions = append(pipeline.Actions, sshDebug()...)
```

Comment:
This should probably go after the make directories step in case you need to use TMPDIR